### DISCUSSION TITLE
Text not parsed from PDF-document?

### DISCUSSION THREAD

**Discussion by Rapid1898-code** [jsvine/pdfplumber#735] [Ask for help with specific PDFs]
Hello - i have some problems parsing the following pdf-document:
https://mega.nz/file/U7RUFRya#oBy7imAY7nJwt8NfB-UdA_ow0c8b8W8LPH0UGyWv6rM

I try to simply extract the text from this pdf using the following code:

```
import pdfplumber
import os
import sys
if __name__ == '__main__':
  path = os.path.abspath(os.path.dirname(sys.argv[0])) 
  fn = os.path.join(path, "exmpl1.pdf")
  pdf = pdfplumber.open(fn)

  page = pdf.pages[0]
 text = page.extract_text()
  print(text)
```

But the only output i get is:

```
$ python test1.py
y
l
n
o

e
s
u

l
a
n
o
s
r
e
p

r
o
F
```

It seems that only the part on the left side is parsed here.
How can i get the other part of the document?

**Comment by jsvine** [comment]
Hi @Rapid1898-code, and thanks for your interest in this library. Unfortunately, your PDF is mostly just an image of text, rather than containing specific information about the text. You can test this by dragging your cursor over the PDF; only the "For personal use only" text (which pdfplumber does recognize) will be highlighted. You *can* try using OCR (optical character recognition) software to add a text layer back to the PDF, but `pdfplumber` will not work as well with those sorts of PDFs as it does with original-text PDFs.
